redash/query_runner/python.py

```diff
@@ -1,9 +1,44 @@
 import sys
 import json
+import logging
 
 from redash.query_runner import *
 from redash import models
 
+import importlib
+
+logger = logging.getLogger(__name__)
+
+try:
+    from RestrictedPython import compile_restricted
+    from RestrictedPython.Guards import safe_builtins
+
+    enabled = True
+except ImportError:
```

Comment:
OK. Will remove it - unless you want to remove it from the requirements file instead?